src/runtime/eventobject.cs

```diff
@@ -80,7 +80,11 @@ internal bool RemoveEventHandler(IntPtr target, IntPtr handler)
             }
 
             IntPtr hash = Runtime.PyObject_Hash(handler);
-            if (Exceptions.ErrorOccurred() || reg == null)
+            if (Exceptions.ErrorOccurred())
```

Comment:
Would make sense to check if `hash == -1`

Reply:
An IntPtr cannot be -1.  Perhaps IntPtr is not the correct return type for PyObject_Hash?

Reply:
You can still compare it to `new IntPtr(-1)`.
Ideally we should have C types mirrored for clarity though.

Reply:
Is `nint` the correct type?

Reply:
It is definitely a better replacement in this case (hooray C# 9!). Judging by https://docs.microsoft.com/en-us/cpp/build/common-visual-cpp-64-bit-migration-issues?redirectedfrom=MSDN&view=msvc-160 , it should be of the right size.

Reply:
I have changed PyObject_Hash to use nint.